Libraries/DirectMLX.h

```diff
@@ -2181,6 +2181,59 @@ namespace dml
         return output;
     }
 
+    inline Expression AveragePooling1(
```

Comment:
This works, but I might prefer one two alternatives:
1. Add a "PoolingBuilder" struct that allows setting dilations. If dilations are set it should go to POOLING1; otherwise, it should go to POOLING.
2. Delete the implementation of existing AveragePooling helper and forward to AveragePooling1 with empty dilations. If dilations are empty in the helper, then the helper should create an array of dilations with size 0 for each dimension.